text/0000-enumerable-some.md

```diff
@@ -0,0 +1,56 @@
+- Start Date: 2017-06-29
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+[`any`](https://emberjs.com/api/classes/Ember.Array.html#method_any) has been
+implemented on `Ember.Enumerable` for years, however native javascript arrays have
+`Array.prototype.some` to perform the same task.
+
+This RFC proposes the addition of `Ember.Enumerable#some`, the aliasing of the old `Ember.Enumerable#any` to it,
+and encourage the usage of the first either in the docs or with deprecation to be as aligned as possible
+with native javascript arrays.
+
+# Motivation
+
+Motivation is to stay in line with web standard.
+
+# Detailed design
+
+The first step is to rename `any` to `some` and recreate `any` as simply an alias
```

Comment:
I wanted to leave the deprecation part optional, that is why I mentioned the deprecation as a separate step. In case we decide to implemente and deprecate, `deprecatingAlias` is indeed how I'd implement this.